src/fi_tostr.c

```diff
@@ -748,6 +748,8 @@ ofi_tostr_hmem_iface(char *buf, size_t len, enum fi_hmem_iface iface)
 	CASEENUMSTRN(FI_HMEM_CUDA, len);
 	CASEENUMSTRN(FI_HMEM_ROCR, len);
 	CASEENUMSTRN(FI_HMEM_ZE, len);
+	CASEENUMSTRN(FI_HMEM_NEURON, len);
```

Comment:
Can you split into 2 commit and back port the NEURON one to 1.15.x? 

Thank you!

Reply:
Done. Will back port the neuron one to v1.15.x after it's merged.